src/app/page.js

```diff
@@ -12,22 +19,59 @@ export default function Home() {
     setSearchTerm('');
   };
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div className="z-10 w-full max-w-5xl items-center justify-between font-mono text-sm lg:flex">
-      <form id="search" onSubmit={handleSearch} className="flex">
-        <input
-          type="text"
-          placeholder="Search for content"
-          value={searchTerm}
-          onChange={(e) => setSearchTerm(e.target.value)}
-          className="border font-mono font-bold border-gray-300 rounded-l px-4 py-2 focus:outline-none focus:ring-2 focus:ring-blue-500 w-64"
-        />
-        <button type="submit"
-        className="bg-blue-500 hover:bg-blue-600 text-white font-mono font-bold py-2 px-4 rounded-r focus:outline-none focus:ring-2 focus:ring-lightblue-500"
-        >Search</button>
-      </form>
-        
+    <main className="flex min-h-screen flex-col">
+      <div class="grid grid-cols-4 gap-4 ml-10 absolute top-3 right-0 flex justify-end stroke-cyan-500 hover:stroke-cyan-700">
```

Comment:
Things like I said we don't need them I don't know where you got the motivation to go about these, I explained to you correctly what you should work on but I don't know this ideal you have.